components/general/typography/_typography.scss

```diff
@@ -1,100 +1,67 @@
-h1,
-h2 {
-    font-weight: 800;;
-    margin: 0 0 10px;
+.heading {
     color: $font-color;
-    line-height: 1.4;
-}
-h3,
-h4,
-h5,
-h6 {
-    font-weight: 700;
-    margin: 0 0 10px;
-    color: $font-color;
-    line-height: 1.4;
-}
+    font-family: $font-family;
+    font-weight: 400;
+    margin: 0 0 15px;
+    font-size: 14px;
+    text-transform: uppercase;
 
-h1 {
-    font-size: 24px;
+    &--bodoni-font {
```

Comment:
Nope nope nope
1. Do not define fonts as modifications, especially using the name of a font.
2. This is a base for other projects, why someone might want to use `Bodoni`?